inotify.go

```diff
@@ -80,7 +80,9 @@ func (w *Watcher) Close() error {
 	close(w.done)
 
 	// Wake up goroutine
-	w.poller.wake()
+	if err := w.poller.wake(); err != nil {
+		return fmt.Errorf("poller.wake failed: %w", err)
+	}
```

Comment:
I'm not entirely sure if this is really the correct behaviour; I think that this failing isn't really a big deal in the Close() here?